.github/workflows/tests.yml

```diff
@@ -2,8 +2,9 @@ name: Tests
 
 on:
   push:
+    branches:
+      - main
   pull_request:
```

Comment:
Nit: we can remove this section now, right? (for both of the files)
```suggestion
```

Reply:
we cannot remove this section, because then the checks wouldn't run on PRs!